pkg/collector/corechecks/cluster/orchestrator/collector_bundle.go

```diff
@@ -268,9 +268,6 @@ func (cb *CollectorBundle) Initialize() error {
 			// would prevent us from restarting them again if the check is unscheduled/rescheduled
 			// see https://github.com/kubernetes/client-go/blob/3511ef41b1fbe1152ef5cab2c0b950dfd607eea7/informers/factory.go#L64-L66
 
-			// TODO: right now we use a stop channel which we don't close, that can lead to resource leaks
-			// A recent go-client update https://github.com/kubernetes/kubernetes/pull/104853 changed the behaviour so that
-			// we are not able to start informers anymore once they have been stopped. We will need to work around this. Once this is fixed we can properly release the resources during a check.Close().
 			go informer.Run(cb.stopCh)
```

Comment:
I see stopCh is set above but does it actually close now?

Reply:
Yes, when the check is cancelled, the informers are stoped https://github.com/DataDog/datadog-agent/blob/908d8758893755eaeb88128ecf0f48638878f8e2/pkg/collector/corechecks/cluster/orchestrator/orchestrator.go#L200